src/ClientInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Just a reminder this and the ClientTrait were removed, you should be good to go if you pull in the latest from master.